Makefile

```diff
@@ -56,6 +56,46 @@ endif
 ifneq (${ARGOCD_AGENT_IN_CLUSTER},)
 	./hack/dev-env/restart-all.sh
 endif
+	@echo ""
```

Comment:
JGW: Move to shell script?

JGW: Could this live in one of the existing shell scripts, like `setup-vcluster-env.sh` or `deploy.sh`?